CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Change log for Microsoft365DSC
 
+# UNRELEASED
+ * Fix example in documentation for Update-M365DSCAzureAdApplication
 
 # 1.24.228.1
```

Comment:
nit picking but still missing space :)

Reply:
Always good to have the second pair of eyes 👍